src/main/java/at/hannibal2/skyhanni/features/inventory/FavoritePowerStone.kt

```diff
@@ -0,0 +1,64 @@
+package at.hannibal2.skyhanni.features.inventory
+
+import at.hannibal2.skyhanni.SkyHanniMod
+import at.hannibal2.skyhanni.data.MaxwellAPI
+import at.hannibal2.skyhanni.data.ProfileStorageData
+import at.hannibal2.skyhanni.events.GuiContainerEvent
+import at.hannibal2.skyhanni.events.InventoryUpdatedEvent
+import at.hannibal2.skyhanni.skyhannimodule.SkyHanniModule
+import at.hannibal2.skyhanni.utils.InventoryUtils
+import at.hannibal2.skyhanni.utils.ItemUtils.name
+import at.hannibal2.skyhanni.utils.KeyboardManager
+import at.hannibal2.skyhanni.utils.LorenzColor
+import at.hannibal2.skyhanni.utils.LorenzUtils
+import at.hannibal2.skyhanni.utils.RenderUtils.highlight
+import at.hannibal2.skyhanni.utils.StringUtils.removeColor
+import net.minecraftforge.fml.common.eventhandler.SubscribeEvent
+
+@SkyHanniModule
+object FavoritePowerStone {
+
+    private val config get() = SkyHanniMod.feature.inventory
+    private val storage get() = ProfileStorageData.profileSpecific
+
+    private var highlightedSlots = mutableSetOf<Int>()
+
+    @SubscribeEvent
+    fun onBackgroundDrawn(event: GuiContainerEvent.BackgroundDrawnEvent) {
+        if (!isEnabled()) return
+        if (!MaxwellAPI.isThaumaturgyInventory(InventoryUtils.openInventoryName())) return
```

Comment:
out of scope of the pr, but we should not expose regex check functions outside, rather cache the value in isInventory boolean. no need to test this regex each frame

Reply:
I had done it before, but couldn't figure a way to fix a bug caused by it. If you can, please test what happens when you open the menu, go to tunings and come back to the main menu (without reopening it). The render used to not work in that situation, I don't know if it'll happen again.